data-es6.js

```diff
@@ -5701,6 +5963,36 @@ exports.tests = [
         iojs:        true,
       },
     },
+    'Promise.race supports iterables': {
+      exec: function () {/*
+        var fulfills = Promise.race(new Set([
```

Comment:
`Promise` methods do not depend on `Set`. For  generic iterables  available `__createIterableObject`.